src/newLaunch/permissoned.html

```diff
@@ -0,0 +1,23 @@
+<template bindable="checked, toggle, disabled">
+  <div class="infoInput">
+    <div class="labeledQuestion">
+      <label for="geoBlock">Set Permission-less SEED</label>
```

Comment:
Please rename the `geoBlock` to something more descriptive. Maybe `permissionedToggle`?
Though, the best could be a reusable component for both...